### ISSUE TITLE
Links do not have a discernible name

### ISSUE THREAD

**Issue by alexdor** [ossn/fixme#19]
Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users. [Learn more](https://dequeuniversity.com/rules/axe/2.2/link-name?application=lighthouse) 

Failing Elements
```
<a class="nav-link" target="_blank" to="https://twitter.com/fixmeparser" href="https://twitter.com/fixmeparser">
```

**Comment by tabshaikh** [comment]
@alexdor So the change to make in this is that to make the twitter icon highlighted

**Comment by tabshaikh** [comment]
@alexdor can you give some useful place to begin with in this

**Comment by alexdor** [comment]
hi @tabshaikh , in order to resolve this you should do something like this https://stackoverflow.com/a/51715544 , let me know if you have any questions : )

**Comment by alexdor** [comment]
Thanks for the reminder, just did : )

**Comment by tabshaikh** [comment]
@alexdor In this issue the twitter icon now has code something like this
![twiiter](https://user-images.githubusercontent.com/31198893/47225585-10079b80-d3dc-11e8-8dc3-91e0db91d051.png)
so the aria-label is there, Also i could not find the exact place where the failing element you pointed out in the issue is. I think the aria label works only in <a></a> tag but as we have used it in <outboundlink> therefore it doesnt work
